SU2_CFD/src/solvers/CRadP1Solver.cpp

```diff
@@ -282,10 +282,15 @@ void CRadP1Solver::BC_Isothermal_Wall(CGeometry *geometry, CSolver **solver_cont
   /*--- Get the specified wall emissivity from config ---*/
   Wall_Emissivity = config->GetWall_Emissivity(Marker_Tag);
 
+  /*--- Validate emissivity is in physical range [0,1] ---*/
+  if (Wall_Emissivity < 0.0 || Wall_Emissivity > 1.0) {
+    SU2_MPI::Error("Wall emissivity must be in range [0,1].", CURRENT_FUNCTION);
+  }
```

Comment:
Validation should be done in CConfig PostProcessing, not in the solver, so that as much as possible the settings can be checked in dry-run mode.
```suggestion
```